docs/user/FitTutorial.rst

```diff
@@ -0,0 +1,279 @@
+    fit_omega = result.fit.rate["omega"]
+
+The resulting fit contains several parts.
+
+-  ``fit`` - This has the same structure as the ``model_variables``
+   above.
+-  ``prior_residuals`` - These are residuals and lagrange values for all
+   priors.
+-  ``data_residuals`` - Residuals on the input data uncertainty.
+-  ``fit_data`` - The data that was included in the fit.
+-  ``excluded_data`` - Which data was excluded from the fit, due to
+   being held out or having covariates that are outliered.
+
+The value residuals should be small, unless we intentionally skewed the
+other-cause mortality with ``mess_factor``.
+
+.. code:: python
+
+    value_residuals = list()
+    for age, time in fit_omega.age_time():
+        value_residuals.append([age, float(fit_omega[age, time] - omega[age, time])])
+    # Makes an age column and a time column.
+    value_residuals = np.array(value_residuals).T
+
+.. code:: python
+
+    def plot_mortality(continuous_mortality, result, data, value_residuals):
+        fig = plt.figure(tight_layout=True)
+        gs = gridspec.GridSpec(2, 2)
+    
+        ax = fig.add_subplot(gs[0, 0])
+        x = np.linspace(0, 120, 241)
+        slabel, = ax.plot(x, continuous_mortality(x), label="siler")
+        omega_fit = result.fit.rate["omega"]
+        flabel, = ax.plot(x, omega_fit(x, 2000), label="fit")
+        mtother_data = data[data.integrand == "mtother"]
+        dlabel = ax.scatter(mtother_data.age_lower.values, mtother_data["mean"].values, label="data")
+        ax.legend(handles=[slabel, flabel, dlabel])
+        ax.set_xlim([0, 70])
+        ax.set_ylim([0, .2])
+        ax.set_xlabel("age")
+        ax.set_ylabel("rate")
+    
+        ax01 = fig.add_subplot(gs[0, 1])
+        ax01.scatter(value_residuals[0], value_residuals[1])
+        ax01.set_xlabel("age")
+        ax01.set_ylabel("absolute residual")
+    
+        ax10 = fig.add_subplot(gs[1, 0])
+        xkids = np.linspace(0, 20, 50)
+        slabel, = ax10.plot(xkids, mortality(xkids), label="siler")
+        # olabel, = ax10.plot(xkids, omega(xkids, 2000), label="omega")
+        omega_fit = result.fit.rate["omega"]
+        flabel, = ax10.plot(xkids, omega_fit(xkids, 2000), label="fit")
+        ax10.legend(handles=[slabel, flabel])
+        # ax.set_xlim([0, 70])
+        # ax.set_ylim([0, .2])
+        ax10.set_xlabel("age")
+        ax10.set_ylabel("rate")
+    
+        ax11 = fig.add_subplot(gs[1, 1])
+        ax11.scatter(value_residuals[0][:5], value_residuals[1][:5])
+        ax11.set_xlabel("age")
+        ax11.set_ylabel("absolute residual")
+    
+        plt.show()
+
+.. code:: python
+
+    plot_mortality(mortality, result, data, value_residuals)
+
+
+.. parsed-literal::
+
+    /home/adolgert/.local/share/virtualenvs/cascade-f6I11cmg/lib/python3.6/site-packages/matplotlib/figure.py:2359: UserWarning: This figure includes Axes that are not compatible with tight_layout, so results might be incorrect.
```

Comment:
You probably didn't mean to have this here...

Reply:
Hah! Matplotlib is hard to wrangle.